src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatibilityAnalyzer.cs

```diff
@@ -285,8 +285,7 @@ private static bool PlatformAnalysisAllowed(AnalyzerOptions options, Compilation
             string tfmVersion = options.GetMSBuildPropertyValue(MSBuildPropertyOptionNames.TargetFrameworkVersion, compilation) ?? "";
 
             if (tfmIdentifier.Equals(NetCoreAppIdentifier, StringComparison.OrdinalIgnoreCase) &&
-                tfmVersion.StartsWith("v", StringComparison.OrdinalIgnoreCase) &&
-                Version.TryParse(tfmVersion[1..], out var version) &&
+                Version.TryParse(tfmVersion.TrimStart(['v', 'V']), out var version) &&
```

Comment:
Worth noting that the reason this is `TrimStart(['v', 'V'])` and not `TrimStart("vV")` is because analyzers are .NET Standard 2.0 and the latter API returns a `ROSpan<char>`.

Since we're .NET Standard 2.0, there is no `Version.TryParse(ROSpan<char>, out Version)` API and we have to convert back to string anways. So I went with what I thought was the shorter/simpler option here.